dotnet-client-libraries/IntroductionToWolverineLibrary/IntroductionToWolverineLibrary/Program.cs

```diff
@@ -0,0 +1,44 @@
+using IntroductionToWolverineLibrary.Models;
+using Wolverine;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+builder.Services.AddControllers();
+builder.Host.UseWolverine( x =>
+{
+    x.PublishAllMessages().ToLocalQueue("local-queue");
+});
+
+var app = builder.Build();
+
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+}
+
+app.UseHttpsRedirection();
+
+app.MapPost("/newOrder", async (NewBookOrder newOrder, IMessageBus bus) => await bus.InvokeAsync(newOrder))
+    .WithName("NewBookOrder")
+    .WithOpenApi();
+
+app.MapPost("/newOrderReply", async (NewBookOrder newOrder, IMessageBus bus) => await bus.InvokeAsync<string>(newOrder))
```

Comment:
Same comment for this "new" prefix is not required

Reply:
Renamed to 'orderReply'.